e2e/framework/testserver.go

```diff
@@ -346,6 +346,21 @@ func (t *TestServer) GenerateGenesis() error {
 	blockGasLimit := strconv.FormatUint(t.Config.BlockGasLimit, 10)
 	args = append(args, "--block-gas-limit", blockGasLimit)
 
+	// add base fee
+	if t.Config.BaseFee != 0 {
+		args = append(args, "--base-fee", *types.EncodeUint64(t.Config.BaseFee))
+	}
+
+	// add burn contracts
+	if len(t.Config.BurnContracts) != 0 {
+		for block, addr := range t.Config.BurnContracts {
+			args = append(args, "--burn-contract", fmt.Sprintf("%d:%s", block, addr))
+		}
+	} else {
+		// london hardfork is enabled by default so there must be a default burn contract
+		args = append(args, "--burn-contract", "0:0x0000000000000000000000000000000000000000")
```

Comment:
Shouldn't this be the default value?

Reply:
Not sure, I would explicitly specify the contract address in order to avoid confusions

Reply:
What happens if the user do not pass the `--burn-contract` param?

Reply:
The map is going to be empty and zero address is going to be used